packages/web-components/fast-foundation/src/combobox/combobox.template.ts

```diff
@@ -44,13 +48,13 @@ export function comboboxTemplate(
                         @keyup="${(x, c) => x.keyupHandler(c.event as KeyboardEvent)}"
                         ${ref("control")}
                     />
-                    <div class="indicator" part="indicator" aria-hidden="true">
-                        <slot name="indicator">
-                            ${options.indicator || ""}
-                        </slot>
-                    </div>
                 </slot>
                 ${endSlotTemplate(options)}
+                <div class="open-icon" part="open-icon" aria-hidden="true">
+                    <slot name="open-icon">
+                        ${options.openIcon || openIcon}
```

Comment:
+1 to Rob's feedback on how to handle these SVG's. If the concern is having something for a given component *by default*, such as checkbox or radio, I'd prefer a simple span element as default slotted content to a baked in SVG. Far less cost, easy enough to style using :before/:after, using css, etc.